src/main/resources/templates/MainPage/main_page.html

```diff
@@ -1,132 +1,128 @@
+               th:href="@{'/product/'+ ${product.productId}}" style="width:20%" onmouseover="this.style.backgroundColor='#F0FFFF'"
+               onmouseout="this.style.backgroundColor='#FFFAFA'">
+                <div class="d-flex w-100 justify-content-between row">
+                    <div id="photo" class="pt-3 column" role="tabpanel"
+                         aria-labelledby="photo-tab">
+                            <div th:unless="${#strings.isEmpty(product.photo)}">
+                                <img th:src="${product.photo}"
+                                     id="myPhoto"
+                                     width="60"
+                                     height="auto">
+                            </div>
+                    </div>
+                    <div>
+                        <p class="m-0" th:text="${product.productName}"></p>
+                        <p class="m-0" th:text="${product.price} + ' zł'"></p>
+                    </div>
+                </div>
+            </a>
         </div>
+    </span>
+    <div class="row m-1 d-flex justify-content-center">
+        <form class="card p-4 mb-3 col-lg-2 h-50" th:action="@{/}">
 
-        <span th:if="${top10List != null}">
-            <h1> Top 10 produktow </h1>
+            <h3 class="me-auto">Filtry</h3>
+            <div class="d-flex flex-wrap p-0 mb-2">
+                <label class="me-3 mb-1">Cena do:</label>
+                <input class="w-100" type="text" th:field="${productSearchCriteria.searchedPriceLimit}"/>
+            </div>
+            <div class="d-inline-flex p-0 mb-1">
+                <input type="checkbox" class="m-2" th:field="${productSearchCriteria.showOnlyAvailable}"
+                       th:value="true">
+                <label>Tylko dostepne </label>
+            </div>
+            <ul class="d-inline-flex p-0 mb-1" th:each="category : ${categoryList}">
+                <div>
+                    <input class="m-2" type="checkbox" th:field="${productSearchCriteria.searchedCategories}"
+                        th:value="${category.categoryName}">
+                    <label th:text="${category.categoryName}"></label>
+                </div>
+            </ul>
+            <div class="row mt-3">
+                <label class="mb-1">Czego szukasz?</label>
+                <input class="mb-1" type="text" th:field="${productSearchCriteria.searchedPhrase}"/>
+                <button type="submit" class="btn btn-primary mb-1">Wyszukaj</button>
+                <a th:href="@{'/?productName='}" class="btn btn-primary mb-1">Wyczysc</a>
+            </div>
+        </form>
+        <div class="card p-4 mb-3 me-0 col-lg-9 mx-auto">
             <table class="table table-striped table-bordered">
                 <thead class="thead-dark">
                 <tr>
-                    <th>Produkty</th>
-                    <th>Cena</th>
-                    <th>Ilosc</th>
+                    <th>Sortowanie -></th>
```

Comment:
Why '->' is here?

Reply:
It's just pointer, where the soriting opions are.

Reply:
Aaaaa